sdk/deepbook/src/deepbook_sdk.ts

```diff
@@ -176,99 +219,152 @@ export class DeepBook_sdk {
 		const txb = new TransactionBlock();
 		const withdraw = txb.moveCall({
 			typeArguments: [token1, token2],
-			target: `dee9::clob::withdraw_quote`,
-			arguments: [txb.object(`${poolId}`), txb.pure(quantity), txb.object(`${accountCap}`)],
+			target: `${PACKAGE_ID}::${MODULE_CLOB}::withdraw_quote`,
+			arguments: [txb.object(poolId), txb.pure(quantity), txb.object(accountCap)],
 		});
 		txb.transferObjects([withdraw], txb.pure(currentAddress));
-		txb.setGasBudget(this.gasBudget);
 		return txb;
 	}
 
 	/**
 	 * @description: swap exact quote for base
 	 * @param token1 Full coin type of the base asset, eg: "0x3d0d0ce17dcd3b40c2d839d96ce66871ffb40e1154a8dd99af72292b3d10d7fc::wbtc::WBTC"
 	 * @param token2 Full coin type of quote asset, eg: "0x3d0d0ce17dcd3b40c2d839d96ce66871ffb40e1154a8dd99af72292b3d10d7fc::usdt::USDT"
+	 * @param client_order_id an id which identify who make the order, you can define it by yourself, eg: "1" , "2", ...
+	 * @param poolId Object id of pool, created after invoking createPool, eg: "0xcaee8e1c046b58e55196105f1436a2337dcaa0c340a7a8c8baf65e4afb8823a4"
+	 * @param quantity Amount of quote asset to swap in base asset
+	 * @param is_bid true if the order is bid, false if the order is ask
+	 * @param baseCoin the objectId of the base coin
+	 * @param quoteCoin the objectId of the quote coin
+	 * @param currentAddress: current user address, eg: "0xbddc9d4961b46a130c2e1f38585bbc6fa8077ce54bcb206b26874ac08d607966"
+	 * @param accountCap Object id of Account Capacity under user address, created after invoking createAccount, eg: "0x6f699fef193723277559c8f499ca3706121a65ac96d273151b8e52deb29135d3"
+	 */
+	public place_market_order(
+		token1: string,
+		token2: string,
+		client_order_id: string,
+		poolId: string,
+		quantity: number,
```

Comment:
Let's have an e2e test where the `quantity` is respected and not disregarded even if we pass in a coin of a larger value.